student-store-ui/src/components/Home/Search.jsx

```diff
@@ -0,0 +1,35 @@
+import "./Search.css"
+
+export default function App(props) {
```

Comment:
nit: Looks like you forgot to rename App -> Search! The functionality will be the same, but this is the name that will appear in stack traces, so it can make debugging confusing.